app/src/main/java/com/github/polybooks/LoginActivity.kt

```diff
@@ -91,16 +87,24 @@ class LoginActivity : AppCompatActivity() {
     }
 
     private fun updateUI(user: FirebaseUser?) {
+        if(user != null) {
+            val intent = Intent(this, UserProfileActivity::class.java).apply {
+                putExtra(EXTRA_MESSAGE, user.displayName)
```

Comment:
The present of extra parameter should be tested : 
https://developer.android.com/reference/androidx/test/espresso/intent/matcher/IntentMatchers#hasextrawithkey

Reply:
I totally agree but this function is called after the completion of an external task (Google authentification) so I still find it very hard to test without creating some security issues... Nevertheless, this function will be tested by my next PR which call this function after email/password authentication. I am still trying though to catch a possible response from Google which would allow me to test it more precisely...